packages/statemanager/test/stateManager.storage.spec.ts

```diff
@@ -14,14 +14,15 @@ import { DefaultStateManager } from '../src/index.js'
 
 import { createAccount } from './util.js'
 
+const isBrowser = new Function('try {return this===window;}catch(e){ return false;}')
 describe('StateManager -> Storage', () => {
   for (const storageCacheOpts of [
     { deactivate: false },
     { deactivate: true },
     { deactivate: false, size: 0 },
   ]) {
     for (const prefixStorageTrieKeys of [false, true]) {
-      it(`should dump storage`, async () => {
+      it.skipIf(isBrowser() === true)(`should dump storage`, async () => {
```

Comment:
(why is this added here? :thinking: )

Reply:
`dumpStorage` crashes in browser because the trie stream issues we have.